config/config.toml

```
@@ -70,3 +70,7 @@ name = "manifest"
 
 [services.presentation]
 name = "presentation"
+
+[services.webhook]
+name = "webhook"
+webhook_timeout_in_seconds = 10
```

Thoughts on using a string here and parse using the `time.ParseDuration` function? 
```suggestion
webhook_timeout = 10s
```

---

perfect